docs/assets/css/asciinema-player.css

```diff
@@ -0,0 +1,2563 @@
+.asciinema-player-wrapper {
```

Comment:
Following the instructions in https://github.com/asciinema/asciinema-player

I tried something like

```
<link rel="stylesheet" href="https://github.com/asciinema/asciinema-player/releases/download/v2.6.1/asciinema-player.css" />
<link rel="stylesheet" href="{{ '/assets/css/asciinema-player-custom.css' | relative_url }}" />
<script async src="https://github.com/asciinema/asciinema-player/releases/download/v2.6.1/asciinema-player.js"></script>
```

which didn't seem to work.

Reply:
Hmm, I guess the instructions do say to add them to your sites assets. It's likely that the references you have above don't work because they're redirects.

Reply:
https://github.com/asciinema/asciinema-server/issues/256 suggests there aren't any good options. So in this case I guess let's include them in site assets.